model_eval.py

```diff
@@ -0,0 +1,40 @@
+from langchain import LLMChain, OpenAI, Cohere, HuggingFaceHub, PromptTemplate
+from langchain.model_laboratory import ModelLaboratory
+
+llms = [
+    OpenAI(temperature=0),
+    Cohere(model="command-xlarge-20221108", max_tokens=20, temperature=0),
+    HuggingFaceHub(repo_id="google/flan-t5-xl", model_kwargs={"temperature": 1}),
+]
+
+model_lab = ModelLaboratory.from_llms(llms)
+
+print(model_lab.compare("What color is a flamingo?"))
+
+prompt = PromptTemplate(
+    template="What is the capital of {state}?", input_variables=["state"]
+)
+model_lab_with_prompt = ModelLaboratory.from_llms(llms, prompt=prompt)
+
+print(model_lab_with_prompt.compare("New York"))
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `ModelLaboratory.from_llms` method is called twice with different parameters. This could lead to unnecessary memory usage and potential confusion about which instance of `model_lab` should be used. Consider using different variable names for each instance to improve readability and maintainability.
```diff
- model_lab = ModelLaboratory.from_llms(llms)
+ model_lab_without_prompt = ModelLaboratory.from_llms(llms)

  print(model_lab.compare("What color is a flamingo?"))

  prompt = PromptTemplate(
      template="What is the capital of {state}?", input_variables=["state"]
  )
- model_lab_with_prompt = ModelLaboratory.from_llms(llms, prompt=prompt)
+ model_lab = ModelLaboratory.from_llms(llms, prompt=prompt)
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->